charts/netbox/values.yaml

```
@@ -742,6 +742,19 @@ terminationGracePeriodSeconds:
 ##         "app.kubernetes.io/name": netbox
 ##
 topologySpreadConstraints: []
+## @section Pod disruption budget
+## Configure PodDisruptionBudget for NetBox web pods
+## @param pdb.enabled Enable PodDisruptionBudget for NetBox web pods
+## @param pdb.minAvailable Minimum number/percentage of pods that must be available
+## @param pdb.maxUnavailable Maximum number/percentage of pods that can be unavailable
+## If neither minAvailable nor maxUnavailable is set, a sensible default is applied:
+## - Single replica: minAvailable: 1
+## - Multiple replicas or HPA enabled: minAvailable: 50%
+##
+pdb:
+  enabled: false
+  # minAvailable: 1
+  # maxUnavailable: 1
```

When the value is described in the schema, it is better to add default values to avoid misuse, at least to some extent.

```suggestion
  minAvailable: 0
  maxUnavailable: 0
```